### ISSUE TITLE
Recover and document Section Vd six-batch optimization initialization

### ISSUE THREAD

**Issue by mstoelzle** [tud-phi/soromox#154]
## Context

[PR #68](https://github.com/tud-phi/soromox/pull/68) added six-batch MATLAB result archives for the Section Vd control-gain comparison, but did not include the process that produced those batches. The earlier MATLAB script in that history only loads and plots existing results; it does not define or execute the batch optimization.

## Information needed

Please recover and document:

- the six initial `Kp`, `Ki`, and `Kd` gain sets for each controller;
- all random seeds and the randomization or sampling procedure;
- the exact execution method used to produce each run (scripts, commands, environment, and whether runs were independent or vectorized);
- how per-run histories and best trajectories were selected and aggregated;
- the intended batch archive schema, including array axes and the relationship between every recorded loss and gain state.

This should be coordinated with [#128](https://github.com/tud-phi/soromox/issues/128) (best-batch selection) and [#129](https://github.com/tud-phi/soromox/issues/129) (loss/gain history alignment). Once the provenance is documented, a separate change can restore reproducible six-batch optimization.

**Comment by mstoelzle** [comment]
@Michele-Martini, while tracing the new PR #149 results, we found another provenance issue that appears directly related to this issue.

### Suspect legacy loss values

The generator committed in PR #53 defines the loss as

```text
L = 1e3 ∫₀²⋅⁵ ||e||² dt + 5e3 ∫₂⋅⁵⁵ ||e||² dt
```

and appears to be a three-iteration, single-start workflow. PR #68 later added MAT files containing 100 iterations across six starts, without committing the code that generated those batches.

More importantly, the losses stored in those MAT files are inconsistent with their saved trajectories:

- Legacy collocated batch-0 initial trajectory:
  - stored loss: `579.1`
  - rescored using the committed objective: `6049.8`
- Legacy best collocated trajectory:
  - stored loss: `255.7`
  - rescored using the committed objective: `2415.5`

Across the legacy batches, the discrepancy is approximately one order of magnitude. The new PR #149 trajectory losses are `5590.7 → 2428.9`, which are consistent with the rescored legacy trajectories. This suggests that the legacy generator may have used `1e2/5e2` weights, divided the recorded losses by ten, or otherwise applied an undocumented normalization.

Could you please check whether the original six-batch generator or related notes still exist and confirm:

- the actual loss weights used to produce the MAT files;
- whether the loss was rescaled before serialization;
- whether the plotted losses received any additional normalization;
- whether the saved initial/best trajectories were reevaluated using different gain states from those associated with the recorded loss?

### Loss definition and plotting

The current approach also needs reconsideration independently of the provenance problem:

- Both optimizers multiply the objective by an arbitrary global factor of `1e3`.
- The second half of the rollout receives an additional relative weight of five.
- The collocated plot divides the stored loss by `100`.
- The synergistic plot displays its stored loss without that division.
- The collocated objective sums heterogeneous PCS strain coordinates without documented physical normalization.
- The synergistic objective uses a different task-space error scale, so its numerical loss is not directly comparable to the collocated loss.

As a result, “Loss” in the figure is neither the actual optimization objective nor a consistently normalized metric across the two controllers.

The global loss multiplier is also entangled with optimizer behavior because the gradients pass through global-norm clipping before Yogi. We should not treat loss scaling as an implicit replacement for selecting optimizer hyperparameters. Conversely, simply removing `1e3` and multiplying the learning rate by `1e3` would not be equivalent for Yogi and gradient clipping.

I suggest that the follow-up work should:

1. Recover the exact legacy objective and any serialization/plotting normalization.
2. Revisit and tune the collocated controller’s integral-error saturation, using the physically motivated `e_sat = 10 mm` choice from PR #135 as the starting point. Compare it against the recovered legacy behavior and verify its effect on tracking, windup, gradients, and optimized gains.
3. Define a canonical, preferably dimensionless loss using documented characteristic error scales.
4. Retain the `1:5` early/late weighting only if that scientific preference is intentional and documented.
5. Plot the actual canonical loss, without a hidden controller-specific divisor.
6. Tune and report optimizer learning rates, clipping thresholds, gradient norms, and update norms separately from the loss definition.
7. Store the objective definition, normalization scales, saturation configuration, and optimizer metadata in future result archives.
8. Add a regression check that recomputes the saved initial and best losses from their corresponding trajectories.

Until the original generator is recovered, I do not think the legacy MAT loss values should be treated as reproducible reference values.

**Comment by Michele-Martini** [comment]
Can we close this now @mstoelzle ?

**Comment by mstoelzle** [comment]
Yes, closing this as completed following the merge of #204. The PR restores reproducible six-start optimization for Section Vd, records initialization and optimizer provenance with aligned per-start histories, and uses a shared dimensionless objective with validated, rescored trajectories.
